EventsExpress/ClientApp/src/containers/inventory-item.js

```diff
@@ -124,7 +133,9 @@ class InventoryItemWrapper extends Component {
                 <VisitorEditItemForm
                     onSubmit={this.onWillTake}
                     onCancel={this.onCancel}
-                    alreadyGet={alreadyGet - (this.state.isWillTake ? 0 : usersInventories.data.find(e => e.inventoryId === item.id).quantity || 0)}
+                    alreadyGet={alreadyGet - (usersInventories.data.find(e => e.userId === user.Id && e.inventoryId === item.id) === undefined
```

Comment:
Let's move this to a separate function - since it is quite complex and hard to read here. You can than use proper variables instead of doint the list traversal twice.